website/docs/api/large-language-models.mdx

```diff
@@ -752,6 +752,24 @@ supports `.yml`, `.yaml`, `.json` and `.jsonl`.
 path = "textcat_examples.json"
 ```
 
+If you want to perform few-shot learning with a binary classifier (i. e. a text
+either should or should not be assigned to a given class), you can provide
+positive and negative examples with the POS/NEG label. An example for spam
```

Comment:
Should we clarify that the "actual" label, the one referring to "POS", is the one defined in the "labels" argument of the task?

Reply:
Done in https://github.com/explosion/spaCy/pull/13051/commits/f6d9e5c4dfe6c70ecfc868906949e1a0d85d15b3.